sdk/confidentialledger/confidential-ledger-rest/test/public/historicalRangeQuery.spec.ts

```diff
@@ -49,47 +49,26 @@ describe("Range query should be successful", function () {
         throw result.body;
       }
     }
-  });
-
-  it("should audit 2000 entries", async function () {
-    const modulus = 5;
-    const numMessagesSent = 2001;
 
-    let correctEntries: string[] = [];
+    const ledgerEntries = await client
+      .path("/app/transactions")
+      .get({ queryParameters: { collectionId } });
 
-    for (let i = 0; i < modulus; i += 1) {
-      const getLedgerEntriesParams = { queryParameters: { collectionId: "" + i } };
-      // get ledger entries for each collection
-      const ledgerEntries = await client.path("/app/transactions").get(getLedgerEntriesParams);
+    if (isUnexpected(ledgerEntries)) {
+      throw ledgerEntries.body;
+    }
 
-      if (isUnexpected(ledgerEntries)) {
-        throw ledgerEntries.body;
-      }
+    const items = paginate(client, ledgerEntries).byPage({ maxPageSize: 20 });
```

Comment:
```suggestion
    const items = paginate(client, ledgerEntries).byPage({ maxPageSize: messagesToSend });
```

Reply:
from what I can tell, the service doesn't actually support any kind of page size parameter: https://learn.microsoft.com/en-us/rest/api/confidentialledger/list-ledger-entries/list-ledger-entries?tabs=HTTP

Reply:
interesting, but it worked 😂

Reply:
Seems like waiting for the transactions to be committed has resolved this